apps/backend/src/libs/hooks/debouncer/get-request-debouncer.hook.ts

```diff
@@ -0,0 +1,32 @@
+import { type Hook } from "../libs/types/types.js";
+import { DEBOUNCER_CONFIG } from "./libs/constants/constants.js";
+import { RequestLimitError } from "./libs/exceptions/exceptions.js";
+
+const getRequestDebouncer = (requestsLimitPerMinute: number): Hook => {
```

Comment:
let's rename it to something more clear e.g setRateLimit, applyRateLimit
```suggestion
const setRateLimit = (requestsLimitPerMinute: number): Hook => {
```